mailpoet/lib/Newsletter/Sending/ScheduledTasksRepository.php

```diff
@@ -156,6 +156,27 @@ public function findRunningSendingTasks(?int $limit = null): array {
       ->getResult();
   }
 
+  /**
+   * @return ScheduledTaskEntity[]
+   */
+  public function findScheduledSendingTasks(int $limit): array {
```

Comment:
`Sending::getScheduledQueues()` has a default value for `$amount` (`Sending::RESULT_BATCH_SIZE`) and this new method doesn't. I don't it that is necessary, just checking if that was a conscious decision and curious about the reasoning behind it.

Reply:
I didn't want to use a constant from a different class as a default value in the repository. In the end, I decided to unify the limit parameter with a similar method `findRunningSendingTasks` from this repository. This change was added as a new commit.